### PR TITLE
Add Prometheus Alerts

### PR DESCRIPTION
**Pull request by samber**
**[Prometheus Alerts](https://github.com/samber/awesome-prometheus-alerts)**

Curated list of Prometheus alerting rules. Most rules are common to every infrastructures. This is a place to share it !

## Requirements for your pull request

- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by samber** [comment]
Just reviewed #1171 and #1519 ;)

Thanks for you work @sindresorhus !!

**Comment by sindresorhus** [comment]
> * Non-generated Markdown file in a GitHub repo.

File: readme.md [hunk 1]
```diff
@@ -586,6 +586,7 @@
 - [Ad-Free](https://github.com/johnjago/awesome-ad-free#readme) - Ad-free alternatives.
 - [Esolangs](https://github.com/angrykoala/awesome-esolangs#readme) - Programming languages designed for experimentation or as jokes rather than actual use.
 - [Prometheus](https://github.com/roaldnefs/awesome-prometheus#readme) - Open-source monitoring system.
+- [Prometheus-Alerts](https://github.com/samber/awesome-prometheus-alerts) - Prometheus queries and rules for alerting.
```
Comment (sindresorhus) [inline readme.md]: This should be indented with a tab and renamed to just `Alerts` here. See other examples of sub-topics.

**Comment by sindresorhus** [comment]
> * This pull request has a descriptive title in the format `Add Name of List`

**Comment by sindresorhus** [comment]
The Awesome graphic doesn't fit as it's not customized. Either customize it for your purpose by adding your list title and some fitting graphics to it, or drop it.

**Comment by sindresorhus** [comment]
> * **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)

**Comment by sindresorhus** [comment]
The readme heading should be `Awesome Prometheus Alerts`, not `Awesome Prometheus alerting rules`.

**Comment by samber** [comment]
@sindresorhus I fixed all but `Non-generated Markdown file in a GitHub repo.`

Prometheus Alerts are stored in `_data/rules.yml` and then are generated with Jekyll, based on a template, to respect Prometheus syntax.

For exemple:

```
- name: Elastic no new documents
  description: No new documents for 10 min!
  query: 'rate(elasticsearch_indices_docs{es_master_node="false"}[10m]) < 1'
  severity: warning
```

becomes

```
- alert: ElasticNoNewDocuments
  expr: rate(elasticsearch_indices_docs{es_master_node="false"}[10m]) < 1
  for: 30m
  labels:
    severity: warning
  annotations:
    summary: "Elastic no new documents (instance {{ $labels.instance }})"
    description: "No new documents for 10 min!\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
```

**Comment by sindresorhus** [comment]
> Prometheus Alerts are stored in `_data/rules.yml` and then are generated with Jekyll, based on a template, to respect Prometheus syntax.

Then it's unfortunately not suitable for inclusion here.

**Comment by sindresorhus** [comment]
You should get it added to https://github.com/roaldnefs/awesome-prometheus instead.

**Pull request closed without merging**